### ISSUE TITLE
Remove inconsistent labelling for form fields

### ISSUE THREAD

**Issue by ubercj** [e-gineering/movement-challenge-calculator#2]
https://github.com/e-gineering/movement-challenge-calculator/blob/638acc8f6179bd9a9457b26b86f705255f35f961/index.html#L53-L68

This combobox is just one example where an interactive component has a visible text label associated to it, but then also has an `aria-label` attribute. This causes the name that is announced to AT to be inconsistent with what is visible on the screen. This is a violation of [WCAG SC 2.5.3 Label in Name](https://www.w3.org/TR/wcag2mobile-22/#success-criterion-2-5-3-label-in-name)
